atom/browser/api/atom_api_cookies.cc

```diff
@@ -0,0 +1,350 @@
+// Copyright (c) 2015 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "atom/browser/api/atom_api_cookies.h"
+
+#include "atom/common/native_mate_converters/gurl_converter.h"
+#include "atom/common/native_mate_converters/value_converter.h"
+#include "base/bind.h"
+#include "base/time/time.h"
+#include "content/public/browser/browser_context.h"
+#include "content/public/browser/browser_thread.h"
+#include "content/public/browser/web_contents.h"
+#include "native_mate/callback.h"
+#include "native_mate/dictionary.h"
+#include "native_mate/object_template_builder.h"
+#include "net/cookies/cookie_monster.h"
+#include "net/cookies/cookie_store.h"
+#include "net/cookies/cookie_util.h"
+#include "net/url_request/url_request_context.h"
+#include "net/url_request/url_request_context_getter.h"
+
+#include "atom/common/node_includes.h"
+
+using content::BrowserThread;
+
+namespace {
+
+bool GetCookieListFromStore(net::CookieStore* cookie_store,
```

Comment:
Declarations like

```c++
FunctionName(Type par_name1,
    Type par_name2,
    Type par_name3)
```

do not match Google C++ Style Guide:
http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Function_Declarations_and_Definitions

The basic guideline is all parameters should have the same kind of indention.